package.json

```diff
@@ -47,13 +47,7 @@
     "compile:commonjs": {
       "command": "babel -d lib/ src/ && ncp ./src/index.d.ts ./lib/index.d.ts",
       "env": {
-        "NODE_ENV": "commonjs"
-      }
-    },
-    "compile:umd": {
```

Comment:
personally I would move all builds (umd, umd:min, cjs, es) to rollup, but not sure how do u feel about it